### PR TITLE
feat(Zaiko): Add presence

### PR DESCRIPTION
**Pull request by nekok500**
## Description 

Added [Zaiko](https://zaiko.io) presence.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

presence settings

Watching a livestream

Browsing event page

</details>

### CODE REVIEW

File: websites/Z/Zaiko/presence.ts [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+		if (document.querySelector(".item-page")) {
+			// white background pattern e.g. 365423
+			(eventTitle = document
+				.querySelector(
+					".item-page > div > div.container > div > div.my-4.col-lg-7 > div > div > div.mb-2 > h1"
+				)
+				?.textContent.trim()),
+				(eventCover = document.querySelector<HTMLImageElement>(
+					".item-page > div > div.container-fluid > div > figure > img"
+				)?.src);
```
Comment (theusaf) [inline websites/Z/Zaiko/presence.ts:77]: Split these lines with semicolon rather than comma

File: websites/Z/Zaiko/presence.ts [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+		playing: "general.playing",
+		paused: "general.paused",
+		watching: "general.watching",
+		watchingLive: "general.watchingLive",
+		buttonWatchStream: "general.buttonWatchStream",
+		buttonViewPage: "general.buttonViewPage",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let Data: iFrameData;
```
Comment (theusaf) [inline websites/Z/Zaiko/presence.ts:16]: For code consistency, use a camelCase variable name instead

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/Z/Zaiko/presence.ts [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+		buttonWatchStream: "general.buttonWatchStream",
+		buttonViewPage: "general.buttonViewPage",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let Data: IFrameData;
+
+presence.on("iFrameData", async (data: IFrameData) => {
+	Data = data;
+});
```
Comment (theusaf) [inline websites/Z/Zaiko/presence.ts:20]: Clarification: I meant to use the variable name `data` instead of `Data`

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/Z/Zaiko/presence.ts (hunk 3 above)
Comment (nekok500) [reply websites/Z/Zaiko/presence.ts]: Sorry, i fixed it.

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Comment by nekok500** [comment]
thanks for reviewing

**Pull request merged**